### DISCUSSION TITLE
public metadata editor

### DISCUSSION THREAD

**Discussion by Danny-dK** [UtrechtUniversity/yoda#283] [Q&A] [answered]
Will the public metadata editor also be updated to the latest version once updates to the metadata form are made? https://github.com/UtrechtUniversity/yoda-portal/tree/yoda-metadata-editor

**Comment by lwesterhof** [comment, marked as answer]
Yes, but we only want to do that with a new releases and not with developments in between.

**Comment by Danny-dK** [comment]
Thanks. That would be perfect. I was hoping for updates during every major release ;)
